src/vic3_world/countries/vic3_country.h

```diff
@@ -60,7 +61,8 @@ class Country
        character_ids_(std::move(options.character_ids)),
        ig_ids_(std::move(options.ig_ids)),
        puppets_(std::move(options.puppets)),
-       overlord_(options.overlord)
+       overlord_(options.overlord),
+       legitimacy_(options.legitimacy)
```

Comment:
didn't realize this was here. Will split legitimacy off into a separate PR because of how large the institution part has gotten